File: sdkwork-app-sdk-csharp/Models/InviteLinkVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class InviteLinkVO
    {
        public string? CreatedAt { get; set; }
        public string? UpdatedAt { get; set; }
        public string? InviteLink { get; set; }
        public string? InviteCode { get; set; }
        public string? ExpireTime { get; set; }
        public string? QrCode { get; set; }
    }
}

File: sdkwork-app-sdk-typescript/src/types/invite-link-vo.ts
/** 邀请链接响应 */
export interface InviteLinkVO {
  /** 创建时间 */
  createdAt?: string;
  /** 更新时间 */
  updatedAt?: string;
  /** 邀请链接 */
  inviteLink?: string;
  /** 邀请码 */
  inviteCode?: string;
  /** 过期时间 */
  expireTime?: string;
  /** 二维码 */
  qrCode?: string;
}
